src/style.css

```diff
@@ -1,3 +1,102 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  background-color: grey;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  line-height: 60px;
+  height: 60px;
+  margin-right: 50px;
+}
+
+@media (width: 1200px) {
+  .img-link {
+    margin-right: 419px;
+  }
+}
+
+@media (width: 1024px) {
+  .img-link {
+    margin-right: 243px;
```

Comment:
The `margin-right` in the `.img-link` class within media queries contradicts the requirement for the header to stretch the full page width without horizontal margins. Adjust these margins to ensure compliance with the layout requirements.